corporate/views/remote_billing_page.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from typing import Dict, Union
+
+from django.conf import settings
+from django.core import signing
+from django.http import HttpRequest, HttpResponse, HttpResponseRedirect
+from django.urls import reverse
+from django.utils.translation import gettext as _
+from django.views.decorators.csrf import csrf_exempt
+
+from zerver.decorator import zulip_login_required
+from zerver.lib.exceptions import JsonableError, OrganizationOwnerRequiredError
+from zerver.lib.push_notifications import uses_notification_bouncer
+from zerver.lib.remote_server import send_to_push_bouncer
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.lib.validator import check_int
+from zerver.models import UserProfile
+from zilencer.models import RemoteZulipServer
+
+billing_logger = logging.getLogger("corporate.stripe")
+
+
+SIGNED_ACCESS_TOKEN_VALIDITY_IN_SECONDS = 60 * 60 * 24
+
+
+@csrf_exempt
+@has_request_variables
+def remote_server_billing_entry(
+    request: HttpRequest,
+    remote_server: RemoteZulipServer,
+    user_uuid: str = REQ(),
+    realm_id: int = REQ(json_validator=check_int),
+) -> HttpResponse:
+    identity_dict = {
+        "realm_id": realm_id,
+        "user_uuid": user_uuid,
+        "remote_server_uuid": str(remote_server.uuid),
+    }
```

Comment:
Should we declare these as a TypedDict so that the type-checker can know these are intended to be the same type here and when we ready them out?

Reply:
Made this a TypedDict